src/main/java/com/hivemq/cli/commandline/CommandLineProvider.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.hivemq.cli.commandline;
+
+import com.hivemq.cli.ioc.DaggerContextCommandLine;
+import com.hivemq.cli.ioc.DaggerMqttCLI;
+import org.jetbrains.annotations.NotNull;
+import picocli.CommandLine;
+
+public class CommandLineProvider {
```

Comment:
Imho this provider should be handled with Dagger and the color scheme, etc. should be configured in the modules provide methods.